### ISSUE TITLE
Liri on Ubuntu Touch cannot save any data

### ISSUE THREAD

**Issue by ShadowEO** [liri-project/liri-browser#225]
The apparmor template for Liri does not seem to allow it to save any settings or userdata. Attempting to save any settings, bookmarks, etc, and restarting the browser ends up with a browser in it's initial state.

**Comment by tim-sueberkrueb** [comment]
@ShadowEO yes, I'm sorry for this inconvenience. Please note that the Ubuntu Touch version is currently in a very experimental state. 
See https://github.com/liri-project/liri-browser/issues/57 and https://github.com/liri-project/liri-browser/issues/169.

**Comment by ShadowEO** [comment]
Thank you for the quick response Tim, My apologies if it's a duplicate (btw, feel free to close it if it is). That said, the features that do work in Ubuntu Touch work great :) The animations and transitions are great!
